src/js/utils.js

```diff
@@ -43,4 +49,18 @@ export const formatDate = (string) => {
     //Returns the "August 1, 2017" portion
     return formattedDate.slice(0, splitIndex - 1);
   }
+};
+
+
+export const validateKey = (responseObject, key, type, module) => {
```

Comment:
I'd throw the exception in the module, that way the stack trace goes to the right place